internal/seclang/rule_parser.go

```diff
@@ -232,13 +232,24 @@ func (p *RuleParser) ParseDefaultActions(actions string) error {
 		if action.Atype == rules.ActionTypeDisruptive {
 			defaultDisruptive = action.Key
 		}
+		// SecDefaultActions can not contain metadata actions
+		if action.Atype == rules.ActionTypeMetadata {
+			return errors.New("SecDefaultAction must not contain metadata actions: " + actions)
+		}
+		// The transformation none is not suitable to be part of the SecDefaultActions
+		if action.Key == "t" && strings.ToLower(action.Value) == "none" {
```

Comment:
Quoting @fzipi :

> MS source code has: "The transformation none is not suitable to be part of the SecDefaultActions". So we need to exclude t:none from a valid default action. In fact, v2 says in the CHANGES file "The use of tfns are deprecated in SecDefaultAction. Should be forbid in the future.".

I currently denied only the `t:none` transformation, should it the extended to any transformation?

Reply:
I think it is sane to exclude _all_ transformations here.

Reply:
Thank you, done!